frontend/src/metabase/components/ErrorPages/ErrorDiagnosticModal.tsx

```diff
@@ -50,6 +54,11 @@ export const ErrorDiagnosticModal = ({
     );
   }
 
+  // enable "send to Slack" button only for local testing and stats.metabase.com
+  const isInternal =
+    window.location.origin === "http://localhost:3000" ||
+    window.location.origin === "https://stats.metabase.com";
```

Comment:
I don't really like the idea of shipping code that is for internal-use-only. Maybe this is a personal hangup that I should just get over, but I wonder if this is actually a "feature" that we could expose to customers. 🤔

Reply:
yup, I needed to slightly shift the approach, coming with an updated PR soon